src/tools/wasm-split/wasm-split.cpp

```diff
@@ -346,6 +346,7 @@ void splitModule(const WasmSplitOptions& options) {
   ModuleSplitting::Config config;
   setCommonSplitConfigs(config, options);
   config.secondaryFuncs.push_back(std::move(splitFuncs));
+  config.secondaryNames.push_back("deferred");
```

Comment:
Is the plan to remove this eventually?

Reply:
I would be happy to make breaking changes to the existing split workflow (e.g. the name of the secondary module) to minimize differences with the multi-split workflow, but this sounds good for now.